spec/features/case_information_feature_spec.rb

```diff
@@ -1,34 +1,458 @@
+                                    "Scotland or Wales")
+        expectations(probation_service: 'England', tier: 'D', team: 'NPS - England', case_allocation: 'CRC')
+      end
+    end
+
+    context "when submitting form enqueue emails" do
+      before do
+        allow(Nomis::Elite2::UserApi).to receive(:user_details).with("PK000223").and_return(user_details)
+        DeliusImportError.create(nomis_offender_id: nomis_offender_id,
+                                 error_type: DeliusImportError::DUPLICATE_NOMIS_ID)
+
+        signin_user
+        visit new_prison_case_information_path('LEI', nomis_offender_id)
+      end
+
+      it 'does not send email if team_id has not been updated', :raven_intercept_exception,
+         vcr: { cassette_name: :case_information_update_email_not_triggered_when_team_not_changed } do
+        choose_country(country: 'Wales')
+        click_button 'Continue'
+        second_page_form(case_alloc: 'nps', tier: 'b', team_option: 'NPS - Wales')
+        click_button 'Continue'
+
+        visit edit_prison_case_information_path('LEI', nomis_offender_id)
+        choose('case_information_tier_c', visible: false)
+
+        expect {
+          click_button 'Update'
+        }.to change(enqueued_jobs, :size).by(0)
+
+        expect(page).not_to have_css(".notification")
+        expect(page).not_to have_css(".alert")
+        expectations(probation_service: 'Wales', tier: 'C', team: 'NPS - Wales', case_allocation: 'NPS')
+        expect(current_url).to have_content "/prisons/LEI/allocations/#{nomis_offender_id}/new"
+      end
+
+      it 'does not send email if probation_service changed to Scotland or Northern Ireland', :raven_intercept_exception,
+         vcr: { cassette_name: :case_information_update_email_not_triggered_when_probation_scotland_or_ni } do
+        choose_country(country: 'Wales')
+        click_button 'Continue'
+        second_page_form(case_alloc: 'nps', tier: 'b', team_option: 'NPS - Wales')
+        click_button 'Continue'
+
+        visit edit_prison_case_information_path('LEI', nomis_offender_id)
+        choose_country(country: 'Northern Ireland')
+
+        expect {
+          click_button 'Update'
+        }.to change(enqueued_jobs, :size).by(0)
+        expect(page).not_to have_css(".notification")
+        expect(page).not_to have_css(".alert")
+      end
+
+      it 'does not send email when updating team has the same ldu', :raven_intercept_exception,
+         vcr: { cassette_name: :case_information_update_email_not_triggered_when_updated_team_has_same_ldu } do
+        choose_country(country: 'England')
+        click_button 'Continue'
+        second_page_form(case_alloc: 'nps', tier: 'b', team_option: 'NPS - England')
+        click_button 'Continue'
+
+        visit edit_prison_case_information_path('LEI', nomis_offender_id)
+
+        page.execute_script("document.getElementsByName('input-autocomplete')[0].value='NPS - England 3'")
```

Comment:
As this is a feature spec, IMHO we should set js: true and then perform the actual clicks if possible.